charts/gateway-helm/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.serviceMonitor.enabled }}
```

Comment:
Do we want an extra layer of protection with something like `{{ if .Capabilities.APIVersions.Has "monitoring.coreos.com/v1" -}}` ?